pages/profile/[username].tsx

```diff
@@ -1,6 +1,6 @@
 import { BaseContainer } from "@components/Page/BaseContainer";
 import { Helmet } from "react-helmet";
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `useRef` from the React library. This is a necessary change as `useRef` is used later in the code to create a reference to a textarea element.
```diff
- import React, { useEffect, useState } from "react";
+ import React, { useEffect, useRef, useState } from "react";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->